xiilib/_gunicorn/charm.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Flask Charm service."""
+
+import logging
+import pathlib
+import typing
+
+import ops
+import yaml
+from charms.data_platform_libs.v0.data_interfaces import DatabaseRequires, DatabaseRequiresEvent
+from charms.data_platform_libs.v0.s3 import (
+    CredentialsChangedEvent,
+    CredentialsGoneEvent,
+    S3Requirer,
+)
```

Comment:
Is this required? s3 should be a later PR?

Reply:
removed, thanks